### ISSUE TITLE
Type error: Cannot find module 'epubjs' or its corresponding type declarations.

### ISSUE THREAD

**Issue by sbmatch** [spencerwooo/onedrive-vercel-index#669]
<html>
<body>

info  - Checking validity of types...
--
21:48:24.825 | Failed to compile.
21:48:24.825 |  
21:48:24.825 | ./components/previews/EPUBPreview.tsx:3:32
21:48:24.825 | Type error: Cannot find module 'epubjs' or its corresponding type declarations.
21:48:24.825 |  
21:48:24.825 | 1 \| import { FC, useEffect, useRef, useState } from 'react'
21:48:24.825 | 2 \| import { ReactReader } from 'react-reader'
21:48:24.826 | > 3 \| import type { Rendition } from 'epubjs'
21:48:24.826 | \|                                ^
21:48:24.826 | 4 \|
21:48:24.826 | 5 \| import Loading from '../Loading'
21:48:24.826 | 6 \| import DownloadButtonGroup from '../DownloadBtnGtoup'
21:48:24.842 | ELIFECYCLE  Command failed with exit code 1.
21:48:24.855 | Error: Command "pnpm build" exited with 1

</body>
</html>

**Comment by myl7** [comment]
`epubjs` is a dependency of `react-reader`, and it is also included in the lock file:

https://github.com/spencerwooo/onedrive-vercel-index/blob/152380bd057d256bcd6cf3ececefcf5826fecaf9/pnpm-lock.yaml#L1326

And locally I can not reproduce the error.
Maybe you could try `pnpm install` first.
Checking if you are using the latest version may be also helpful.

**Comment by sbmatch** [comment]
> > the module "@fortawesome/fontawesome-common-types" cannot be found
> 
> This should be resolved by #642

some good, it's working fine now, thank you again for your patience

```
╰─$ pnpm build
> onedrive-vercel-index@0.1.0 build /home/kali/temp/onedrive-vercel-index
> next build

info  - Checking validity of types
info  - Creating an optimized production build ...                                                  info  - Creating an info  iinfo  - Creating an optimized production build
info  - Compiled successfully
info  - Collecting page data
info  - Generating static pages (6/6)
info  - Finalizing page optimization

Page                                              Size     First Load JS
┌ ○ / (628 ms)                                    487 B           590 kB
├   /_app                                         0 B             296 kB
├ ○ /[...path] (647 ms)                           531 B           590 kB
├ ○ /404                                          196 B           297 kB
├ λ /api                                          0 B             296 kB
├ λ /api/proxy                                    0 B             296 kB
├ ○ /onedrive-vercel-index-oauth/step-1 (440 ms)  3.5 kB          322 kB
├ ○ /onedrive-vercel-index-oauth/step-2 (490 ms)  1.97 kB         347 kB
└ λ /onedrive-vercel-index-oauth/step-3           2.41 kB         348 kB
+ First Load JS shared by all                     296 kB
  ├ chunks/framework-0e98bcaf2420af3e.js          42.1 kB
  ├ chunks/main-973fbe287b882966.js               28.7 kB
  ├ chunks/pages/_app-3961b070c1682171.js         224 kB
  ├ chunks/webpack-048d53d5b082155e.js            1.8 kB
  └ css/33778090a30883bc.css                      9.16 kB
```
